apps/website/src/styles/components/Button.ts

```diff
@@ -14,34 +13,53 @@ const Button = {
         paddingRight: "18px !important"
     },
     defaultProps: {
-        colorScheme: "#FFFFFF"
+        colorScheme: "white"
     },
     variants: {
         solid: (props: StyleFunctionProps): SystemStyleObject => {
             const { colorScheme: c } = props
 
+            if (c === "primary") {
+                const bgGradient = "linear(to-r, primary.500, primary.800)"
+                const color = "white"
+
+                return {
+                    bgGradient,
+                    color,
```

Comment:
```suggestion
                    color,
                    transitionDuration:"200ms",
                    transitionTimingFunction: "linear",
```

What about adding something like this here?

Reply:
Maybe do the same with all the other buttons?

Reply:
I like it, I've added them